appveyor.yml

```diff
@@ -3,7 +3,7 @@
 # BSD-style license that can be found in the LICENSE file.
 
 install:
-  - ps: wget https://gsdview.appspot.com/dart-archive/channels/dev/raw/2.0.0-dev.22.0/sdk/dartsdk-windows-x64-release.zip -OutFile dart-sdk.zip
+  - ps: wget https://gsdview.appspot.com/dart-archive/channels/dev/raw/latest/sdk/dartsdk-windows-x64-release.zip -OutFile dart-sdk.zip
```

Comment:
Should be `https://storage.googleapis.com/dart-archive/channels/dev/release/latest/sdk/dartsdk-windows-x64-release.zip`

Reply:
This is the "official" external URI

Reply:
see above

Reply:
At least change the domain. `gsdview.appspot.com` is a weird internal tool that happens to work.

No promises that it will stick around. It adds a layer on to downloading the SDK, etc.

Reply:
Oh, yeah, good idea.  No idea why it was this way in the first place.  Done.